app/models/forem_instance.rb

```diff
@@ -27,4 +27,8 @@ def self.dev_to?
   def self.smtp_enabled?
     (Settings::SMTP.user_name.present? && Settings::SMTP.password.present?) || ENV["SENDGRID_API_KEY"].present?
   end
+
+  def self.private?
```

Comment:
This seemed like it might come in handy in the future and is consistent with how we perform other check, e.g. `local?` and `dev_to?`.